src/zfh.tex

```diff
@@ -348,3 +348,46 @@ \section{Half-Precision Floating-Point Classify Instruction}
 FCLASS & H & 0 & src & 001 & dest & OP-FP  \\
 \end{tabular}
 \end{center}
+
+\section{``Zfhmin'' Standard Extension for Minimal Half-Precision Floating-Point Support}
+
+{\bf Warning! This draft specification may change before being
+accepted as standard by RISC-V International.}
+
+This section describes the Zfhmin standard extension, which provides minimal
+support for 16-bit half-precision binary floating-point instructions.
+The Zfhmin extension is a subset of the Zfh extension, consisting only
+of data transfer and conversion instructions.
+Like Zfh, the Zfhmin extension depends on the single-precision floating-point
+extension, F.
+The expectation is that Zfhmin software primarily uses the half-precision
+format for storage, performing most computation in higher precision.
+
+The Zfhmin extension includes the following instructions from the Zfh
+extension: FLH, FSH, FMV.X.H, FMV.H.X, FCVT.S.H, and FCVT.H.S.
+If the D extension is present, the FCVT.D.H and FCVT.H.D instructions are
+also included.
+If the Q extension is present, the FCVT.Q.H and FCVT.H.Q instructions are
+additionally included.
+
+\begin{commentary}
+FSGNJ.S is used to move half-precision values between floating-point registers.
+\end{commentary}
+
+\begin{commentary}
+Half-precision addition, subtraction, multiplication, division, and
+square-root operations can be faithfully emulated by converting the
+half-precision operands to single-precision, performing the operation
+using single-precision arithmetic, then converting back to half-precision.
+Performing half-precision fused multiply-addition using this method incurs
+a 1-ulp error on some inputs.
```

Comment:
Suggest clarifying that the "error" is only on RNE and RMM (double-rounding is innocuous for directed rounding).

Reply:
@knightsifive Yeah, good point.